### PR TITLE
Add devops pipelines

### PR DESCRIPTION
**Pull request by ElfoLiNk**

### CODE REVIEW

File: .devops/code-review-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  # B) Run unit tests if there is a push or pull request on any branch.
+  - stage: Test
+    dependsOn: []
+    jobs:
+      - job: unit_tests
+        steps:
+        - template: templates/node-job-setup/template.yaml@pagopaCommons      
+
+        - script: |
+            yarn test:coverage
```
Comment (AleDore) [inline .devops/code-review-pipelines.yml:76]: we must add this script into `package.json`

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+              endpointType: 'private'
+              containerInstanceResourceGroup: '$(HEALTHCHECK_CONTAINER_RESOURCE_GROUP_NAME)'
+              containerInstanceVNet: '$(HEALTHCHECK_CONTAINER_VNET)'
+              containerInstanceSubnet: '$(HEALTHCHECK_CONTAINER_SUBNET)'
+
+          - task: AzureAppServiceManage@0
+            inputs:
+              azureSubscription: '$(AZURE_SUBSCRIPTION)'
+              resourceGroupName: '$(WEB_APP_RESOURCE_GROUP_NAME)'
+              webAppName: '$(WEB_APP_NAME)'
```
Comment (AleDore) [inline .devops/deploy-pipelines.yml:181]: AFAIK we don't have staging and production slot by using app service. Right? @pasqualedevita

File: .devops/code-review-pipelines.yml (hunk 1 above)
Comment (ElfoLiNk) [reply .devops/code-review-pipelines.yml:76]: Added all the missing ones

File: .devops/deploy-pipelines.yml (hunk 2 above)
Comment (pasqualedevita) [reply .devops/deploy-pipelines.yml:181]: this new pipeline handle both cases setting these vars in gitops:
UAT_DEPLOY_TYPE
PROD_DEPLOY_TYPE

values:
1. `production_slot`
2. `staging_slot_and_swap`

My opinion, I prefer to have all pipeline with same structure

**Review by pasqualedevita — APPROVED** [review]

**Comment by AleDore** [comment]
I agree @pasqualedevita . @ElfoLiNk Dow we have to align pipelines as suggested by Pasquale?

**Comment by ElfoLiNk** [comment]
I took the latest version so should be already aligned

**Review by AleDore — APPROVED** [review]

**Comment by pasqualedevita** [comment]
yes yes it's latest version

**Pull request merged**